src/components/Button.js

```diff
@@ -0,0 +1,3 @@
+export default function Button() {
+	return <button type="button">auswählen</button>;
+}
```

Comment:
Actually, every PR should only contain components that are used, but we had agreed on this secret button  🤫